backend/inspirehep/disambiguation/tasks.py

```diff
@@ -64,3 +67,21 @@ def disambiguate_signatures(self, clusters):
             disambiguation_assigned_clusters.labels("2+").inc()
             LOGGER.debug("Received cluster with more than 1 author.")
     db.session.commit()
+
+
+@shared_task(ignore_result=False, bind=True)
+def disambiguate_authors(self, record_uuid):
+    record = LiteratureRecord.get_record(record_uuid)
+    if "Literature" not in record["_collections"]:
+        return
+    authors = record.get("authors", [])
+    updated_authors = []
+    for author in authors:
```

Comment:
this is expensive for records with many authors, so I guess you want to restrict to the cases where relevant metadata has changed: either an author has been added/removed (check the `uuid`), or `ids` of an existing author have been changed. Any other modification of the record won't change the linked author (for this MVP, later we will add more metadata). You can do that by comparing with the previous version of the record, similarly to what we're doing for references: https://github.com/inspirehep/inspirehep/blob/7d97abcdd4f2b97d4b5a651d3c776a92d6b0533c/backend/inspirehep/records/api/literature.py#L174-L222